javascript/webpacker_react-npm-module/src/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import _ from 'lodash'
```

Comment:
If we use lodash, then we should only import the functions we use, not the whole library